cmd/podman/commands.go

```diff
@@ -182,6 +182,8 @@ func getDefaultPidsLimit() int64 {
 		cgroup2, _ := cgroups.IsCgroup2UnifiedMode()
```

Comment:
I think we also have to verify that cgroup manager is set to systemd - @giuseppe can you confirm?

Reply:
yes, with cgroupfs rootless is still not able to use cgroups

Reply:
Fixed